src/sentry/static/sentry/app/components/loading/loadingContainer.tsx

```diff
@@ -18,17 +20,21 @@ type Props = {
 
 type MaskProps = {
   isReloading: boolean;
+  maskBackgroundColor: string;
 };
 
 export default function LoadingContainer(props: Props) {
```

Comment:
IMO it's a good idea to be explicit about types, so it'll be better to convert to `React.FC<Props>` type on this old function

Reply:
I'd like to move that to a follow up PR.